CONTRIBUTING.md

```diff
@@ -1,44 +1,63 @@
+If you know a word that should be defined, but – for whatever reason – don’t feel confident writing the definition yourself (or maybe you disagree with an existing defintion), please [open an issue](https://github.com/tatianamac/selfdefined/issues/new). We can then collectively figure out how to best get this word defined.
+
+An issue might also be appropriate if you are unsure about some nuances of a definition you want to define. By opening an issue you start a conversation, and that is always a good thing!
 
-An issue might also be appropriate if you are unsure about some nuances of a definition you want to define.
+If you are new to Github and feel unsure about what an issue should look like, see examples of a [good first issue](https://github.com/tatianamac/selfdefined/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22good+first+issue%22+).
 
 ### Version 2: Pull Requests
 
-You got a definition and want to define it. That’s awesome. Let’s go. Currently, you need to fork this repository and open a PR out of your fork. If that sounds like gibberish to you, Chaser Pettit wrote a [great introduction to the workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962).
+You got a definition and want to define it. That’s awesome. Let’s go. Currently, you need to fork this repository and open a pull request (PR) out of your fork. If that sounds like gibberish to you, Chaser Pettit wrote a [great introduction to the workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962).
 
 #### Working on Words
 
-After you’ve forked the repo, you will need to write a definition. Good times. Hopefully. Always remember, if you are unsure, you can first open an issue or create a PR, include «WIP» in the title, and ask for early feedback.
+After you’ve forked the repo, you will need to write a definition. Good times. Hopefully. Always remember that if you are unsure, you can first open an issue or create a PR, include «WIP» in the title («WIP» stands for «work in progress»), and ask for early feedback.
 
 - Step 1: Create a file named `word.md` in in the [definitions folder](11ty/definitions/) where we store all definitions.
 
   `word` should be the name of the word you are defining. Not `word`. Except you are defining «word». In this case it should be `word`. 
 
-  For this tutorial I will take the word «Obsessive Compulsive Disorder», as it allows me to show some of the features. That means I would create the file `obsessive-compulsive-disorder.md`. As it is already defined, you can have a look at the [finished definition](11ty/definitions/obsessive-compulsive-disorder.md).
+  For this tutorial we will take the word «Obsessive Compulsive Disorder», as it allows us to show some of the features. That means we would create the file named `obsessive-compulsive-disorder.md`. As it is already defined, you can have a look at the [finished definition of «Obsessive Compulsive Disorder»](11ty/definitions/obsessive-compulsive-disorder.md).
```

Comment:
I think we should put "As it is..." onward into a parenthetical ()

Reply:
done